services/celery/celery.py

```diff
@@ -176,5 +176,13 @@ def computer_eth_usdc_volatility():
 def computer_btc_usdc_volatility():
     dydx_order_manager_obj = DydxOrderManager()
     dydx_order_manager_obj.market_volatility(symbol="BTCUSDC")
+
+@app.task(
+    name="set_price_floor",
+)
+def set_price_floor():
+    price_floor_manager_obj = PriceFloorManager()
+    a = price_floor_manager_obj.set_price_floor('ethereum')
```

Comment:
is this function only for setting price floor of **'ethereum'** ?

Reply:
so if you are talking about set_price_floor task than yes and if you talk about price_floor_manager_obj.set_price_floor so this function can be used to set price floor of any asset

Reply:
as of now  we are passing ethereum so it will set floor price of ethereum if we pass the bitcoin than it will set price floor for bitcoin as well